### ISSUE TITLE
Infographic Source Updated

### ISSUE THREAD

**Issue by Mynxee** [bobbyroberts99/Signal-Cartel#8]
I made a few small changes to the infographic. [Updated version here](http://i.imgur.com/n5WeJ4m.jpg).

1. Under **What to Do**, added Astrophotographer which should link to the [EvE-Scout Observatory](http://observatory.eve-scout.com/).
2. Under **Corp Programs**, changed *Private Subreddit* to *Private Subreddit/Wiki* in anticipation of having the Confluence wiki roll out within a week or so (once I get time to add some structure there). 
3. Tweaked spacing and placement of a few items.

**Comment by bobbyroberts99** [comment]
2. Is the confluence wiki different from the [reddit wiki](http://www.reddit.com/r/signalcartel/wiki/index)?

3. I think I got all your tweaks, but let me know if I missed some.

**Comment by Mynxee** [comment]
Yes, the Confluence wiki is a much more robust tool that can be adminned in a granular fashion that will allow us to invite outsiders as collaborators on research projects, as well as other things.

**Comment by Mynxee** [comment]
Under What to Do, the one bullet was changed from Work Exploration sites to Find/Work Exploration Sites. Yours needs to reflect that.
